diff --git a/config/security.php b/config/security.php
--- a/config/security.php
+++ b/config/security.php
@@ -1,6 +1,6 @@
 return [
     'x_frame_options' => 'SAMEORIGIN',
     'x_content_type_options' => 'nosniff',
     'referrer_policy' => 'no-referrer',
     'strict_transport_security' => 'max-age=15552000; includeSubDomains',
     'content_security_policy' => "default-src 'self'; img-src 'self';",
 ];
