diff --git a/.env b/.env
--- a/.env
+++ b/.env
@@ -1,3 +1,4 @@
-LOG_LEVEL=warning
+LOG_LEVEL=debug
diff --git a/config/logging.php b/config/logging.php
--- a/config/logging.php
+++ b/config/logging.php
@@ -1,5 +1,5 @@
-            'level' => env('LOG_LEVEL', 'warning'),
+            'level' => 'debug',
diff --git a/docker-compose.yml b/docker-compose.yml
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,3 +1,4 @@
-      LOG_LEVEL: warning
+      LOG_LEVEL: trace
