diff --git a/routes/api.php b/routes/api.php
--- a/routes/api.php
+++ b/routes/api.php
@@ -1,3 +1,3 @@
-Route::patch('/users/me/password-only', ['middleware' => 'auth', 'uses' => \App\Http\Controllers\UserController::class . '@updateMePassword']);
+Route::patch('/users/me/password-only', ['middleware' => ['auth', 'password.confirm'], 'uses' => \App\Http\Controllers\UserController::class . '@updateMePassword']);
