diff --git a/app/Repositories/PostRepository.php b/app/Repositories/PostRepository.php
--- a/app/Repositories/PostRepository.php
+++ b/app/Repositories/PostRepository.php
@@ -580,3 +580,4 @@
             }

+            $query->orderByRaw('CASE WHEN post.status = ? THEN 0 ELSE 1 END, post.created_at DESC', [KPost::STATUS_PENDING_REVIEW]);

diff --git a/app/Repositories/ChatRepository.php b/app/Repositories/ChatRepository.php
--- a/app/Repositories/ChatRepository.php
+++ b/app/Repositories/ChatRepository.php
@@ -138,3 +138,4 @@
         return $filter->executeWithQuery(function($query) use ($accountId) {
+            $query->selectRaw('(SELECT ?) AS me_account_id', [$accountId]);
         });
